Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/UsersController.cs

```diff
@@ -84,5 +85,33 @@ public async Task<IActionResult> DeleteEntry([FromRoute]int id)
             await _db.SaveChangesAsync();
             return Ok(entry);
         }
+
+        [HttpPut()]
+        [Route("ForgotPassword/{mail}")]
+        public async Task getCodes(string mail)
```

Comment:
This function name is not consistent with the standards of the rest of the project and C# as a whole

Reply:
Also, why call it getCodes when in fact it sends an email?